src/com/owncloud/android/lib/resources/users/GetRemoteUserQuotaOperation.java

```diff
@@ -0,0 +1,119 @@
+package com.owncloud.android.lib.resources.users;
+
+import android.util.Base64;
+import android.util.Log;
+
+import com.owncloud.android.lib.common.OwnCloudBasicCredentials;
+import com.owncloud.android.lib.common.OwnCloudClient;
+import com.owncloud.android.lib.common.OwnCloudCredentials;
+import com.owncloud.android.lib.common.operations.RemoteOperation;
+import com.owncloud.android.lib.common.operations.RemoteOperationResult;
+import com.owncloud.android.lib.common.utils.Log_OC;
+
+import org.apache.commons.httpclient.Header;
+import org.apache.commons.httpclient.HttpClient;
+import org.apache.commons.httpclient.NameValuePair;
+import org.apache.commons.httpclient.methods.GetMethod;
+import org.apache.http.HttpStatus;
+import org.json.JSONObject;
+
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+
+/**
+ * Created by marcello on 11/11/14.
+ */
+public class GetRemoteUserQuotaOperation extends RemoteOperation {
+
+    private static final String TAG = GetRemoteUserNameOperation.class.getSimpleName();
+
+
+    private static final String NODE_OCS = "ocs";
+    private static final String NODE_DATA = "data";
+    private static final String NODE_QUOTA = "quota";
+    private static final String NODE_QUOTA_FREE = "free";
+    private static final String NODE_QUOTA_USED = "used";
+    private static final String NODE_QUOTA_TOTAL = "total";
+    private static final String NODE_QUOTA_RELATIVE = "relative";
+
+    // OCS Route
+    private static final String OCS_ROUTE ="/ocs/v1.php/cloud/users/";
+
+    @Override
+    protected RemoteOperationResult run(OwnCloudClient client) {
+        RemoteOperationResult result = null;
+        int status = -1;
+        GetMethod get = null;
+
+
+        //Get the user
+        try {
+            OwnCloudBasicCredentials credentials = (OwnCloudBasicCredentials) client.getCredentials();
+            String url = client.getBaseUri() + OCS_ROUTE + credentials.getUsername();
+
+            get = new GetMethod(url);
+            get.addRequestHeader(createAuthenticationHeader(credentials));
```

Comment:
This line not necessary, OwnCloudClient applies the credentials. In this point of the code it should be already authenticated.